assignments/function-conversion.js

```diff
@@ -3,28 +3,52 @@
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
+
```

Comment:
if you feel the need to write one liners, can you find a way to make there even more concise?

Reply:
`let carModels = [];

const modelsSorted = inventory.sort((a, b) => {

    let modelA = a.car_model.toLowerCase();
    let modelB = b.car_model.toLowerCase();
    if (modelA > modelB) {
        return 1;
    } else if (modelA < modelB) {
        return -1;
    } return carModels;
});
console.log(modelsSorted); `

Like this?